src/main/java/fr/openmc/core/utils/text/TextManager.java

```diff
@@ -0,0 +1,132 @@
+package fr.openmc.core.utils.text;
+
+import org.bukkit.configuration.file.YamlConfiguration;
+import org.bukkit.configuration.file.FileConfiguration;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import fr.openmc.core.OMCPlugin;
+
+
+public class TextManager {
+
+    private final String defaultLanguage;
+    private final OMCPlugin plugin;
+    private final File translationFolder;
+    private final Map<String, FileConfiguration> loadedLanguages;
+
+    public TextManager(OMCPlugin plugin, File translationFolder, String defaultLanguage) {
+        this.plugin = plugin;
+        this.defaultLanguage = defaultLanguage;
+        this.translationFolder = translationFolder;
+        this.loadedLanguages = new HashMap<>();
+    }
+
+    /**
+     * Returns a string corresponding to the specified path and the language.
+     *
+     * @param path  The path to the text, "default" for default language"
+     * @param language The language of the text
+     */
+    public String getText(String path, String language) {
+        FileConfiguration languageConfig = this.loadedLanguages.get(language);
+                this.loadedLanguages.get(this.defaultLanguage);
+        if (languageConfig != null || language == this.defaultLanguage) {
+            return languageConfig.getString(path, "Missing text for path: " + path);
+        } else {
+            return getText(path);
+        }
+    }
+
+    /**
+     * Returns a string corresponding to the specified path and the language and replaces the given placeholders with the values.
+     *
+     * @param path  The path to the text
+     * @param language The language of the text, "default" for default language"
+     * @param placeholders The placeholders you want to replace in pair with values ("player", player.getName())
+     */
+    public String getText(String path, String language, String... placeholders) {
+        return replacePlaceholders(getText(path, language), placeholders);
```

Comment:
Utilise le mot-clé `this` pour que nous puissions comprendre que tu fais référence à l'instance présente et non à une variable statique ou à un import statique.
```suggestion
        return this.replacePlaceholders(getText(path, language), placeholders);
```